casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -389,6 +389,42 @@ class HashSetCasperTest extends FlatSpec with Matchers {
     } yield result
   }
 
+  it should "handle multi-parent blocks correctly when they operate on stdout" ignore effectTest {
+    def echoContract(no: Int) = s"""new stdout(`rho:io:stdout`) in { stdout!("Contract $no") }"""
```

Comment:
Does this test fail currently? Try not using stdout.

Reply:
Plan was that `stdout` (and other channels used for printing) will not make it to the blockchain. Does this problem appear with other channels as well?